debian/control

```diff
@@ -10,7 +10,7 @@ Package: undistract-me
 Architecture: all
 Section: misc
 Priority: extra
-Depends: ${shlibs:Depends}, ${misc:Depends}, libnotify-bin
+Depends: ${shlibs:Depends}, ${misc:Depends}, libnotify-bin, pulseaudio-utils, sound-theme-freedesktop
```

Comment:
pulseaudio-utils: dependency for paplay.
sound-theme-freedesktop: dependency for sound file.